node/node-core/src/eth/call.rs

```diff
@@ -0,0 +1,41 @@
+use ethers_core::types::{transaction::eip2930::AccessList, Address, Bytes, U256};
+use serde::Deserialize;
+
+/// Call request
+#[derive(Debug, Default, PartialEq, Deserialize, Clone)]
+#[serde(deny_unknown_fields)]
+#[serde(rename_all = "camelCase")]
+pub struct CallRequest {
```

Comment:
Should we be reusing `TransactionRequest` / `TypedTransaction` in some way here instead of repeating the type def?